src/viam/examples/modules/complex/gizmo/api.cpp

```diff
@@ -206,13 +206,12 @@ bool GizmoClient::do_one(std::string arg1) {
     DoOneRequest request;
     DoOneResponse response;
 
-    grpc::ClientContext ctx;
-    set_client_ctx_authority(ctx);
+    auto ctx = make_cc();
 
     *request.mutable_name() = this->name();
     request.set_arg1(arg1);
 
-    const grpc::Status status = stub_->DoOne(&ctx, request, &response);
+    const grpc::Status status = stub_->DoOne(ctx.get(), request, &response);
```

Comment:
If you wanted to avoid needing to write `.get` everywhere, you could make a wrapper type instead of a factory function and give that wrapper a conversion to `ClientContext`:

https://godbolt.org/

Another potential advantage of the wrapper type approach is that if you ever wanted it to start carrying state you would have a place to store that.